src/Compilers/CSharp/Portable/CSharpResources.resx

```diff
@@ -5811,4 +5811,7 @@ To remove the warning, you can use /reference instead (set the Embed Interop Typ
   <data name="ERR_NotBaseOrImplementedInterface" xml:space="preserve">
     <value>'{0}' is not base type or interface of {1}.</value>
   </data>
+  <data name="ERR_RuntimeDoesNotSupportProtectedAccessForInterfaceMember" xml:space="preserve">
+    <value>Target runtime doesn't support 'protected', 'protected internal', or 'private protected' accessibility for an interface member, or a type nested into an interface.</value>
```

Comment:
> or a type nested into an interface [](start = 138, length = 35)

This clause at the end of the diagnostic is confusing to me.  All modern runtimes support types declared inside interfaces.  Do you mean that they aren't permitted with these access levels?  If so, I don't think this clause is helpful, as they are interface members and therefore included in the previous clause.

Also, I would suggest changing "an interface member" to "a member of an interface".

Reply:
Adjusted the message

---
In reply to: [264406631](https://github.com/dotnet/roslyn/pull/34012#discussion_r264406631) [](ancestors = 264406631)